packages/jest-snapshot/src/index.ts

```diff
@@ -211,8 +214,10 @@ const _toMatchSnapshot = ({
 };
 
 const toThrowErrorMatchingSnapshot = function(
+  this: Context,
   received: any,
   testName?: string,
+  // @ts-ignore
```

Comment:
?

Reply:
TS1016: A required parameter cannot follow an optional parameter.

Reply:
That makes sense. Since this function s not exported, you can swap their order so the optional one is last. Or change the optional one to `testName: string | undefined` instead

Reply:
There is export at the end. https://github.com/facebook/jest/blob/cdb6096c2f5884ef8ecbdeaf7439568dadec36fe/packages/jest-snapshot/src/index.ts#L298-L311

I will change to `testName: string | undefined`

Reply:
hah, look at that. We should make this ESM (only a breaking change in `main`). For now, that should be `export =` 🙂